client/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">172.23.49.75</domain>
+    </domain-config>
```

Comment:
I am thinking that is it possible that we allow all the IP addresses from 172.xx.xx.xx? So in the future test, we don't need to modify the code every time we reconnect to a wifi connection.

Reply:
I think we will run this experiment in the same room of the university, and all participant will in the same room, so I don't think we need to change IP for each time.